data/campaigns/Eastern_Invasion/units/Ant_Queen.cfg

```diff
@@ -0,0 +1,57 @@
+#textdomain wesnoth-ei
+[unit_type]
+    id=Ant Queen
+    name= _ "Ant Queen"
+    race=monster
+    image="units/ant/queen-ant.png"
+    profile="portraits/monsters/ant-giant.webp"
+    {DEFENSE_ANIM "units/ant/queen-ant-defend.png" "units/ant/queen-ant.png" hiss.wav }
+    hitpoints=94
+    movement_type=orcishfoot
+    movement=1 # she's been eating too much ant ambrosia
+    experience=100
+    level=3
+    alignment=neutral
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=31
+    usage=fighter
+    description= _ "At the heart of each giant ant burrow lies a queen. Much like their miniature cousins, a well-fed giant ant queen will continually lay eggs, ensuring that the colony never lacks workers for scavenging or soldiers for defense."
+    die_sound=hiss-die.wav
+    undead_variation=spider
+    [abilities]
+        {ABILITY_LEADERSHIP}
+        [dummy]
+            name=_"royalty"
+            description=_"If the ant queen is killed, all of her subjects die."
```

Comment:
This might be convenient from a plot standpoint, but logically it makes no sense whatsoever.

Reply:
It's a common trope: https://tvtropes.org/pmwiki/pmwiki.php/Main/NoOntologicalInertia

Reply:
I don't think that's a good enough justification.